File: reverse-string/reverse-string.cs
public class Solution {
    public void ReverseString(char[] s) {
        var left = 0;
        var right = s.Length-1;
        
        while(left < right)
        {
            (s[left], s[right]) = (s[right], s[left]);
            //var temp = s[left];
            //s[left] = s[right];
            //s[right] = temp;
            
            left++;
            right--;
        }        
    }
}

File: reverse-string/reverse-string.ts
/**
 Do not return anything, modify s in-place instead.
 */
function reverseString(s: string[]): void {
    let left: number = 0;
    let right: number = s.length-1;
    
    while(left < right){
        const temp = s[left];
        s[left] = s[right];
        s[right] = temp;
        left++;
        right--;
    }
};